File: src/host/host.py
import sys
import os

pplib = os.path.join(os.environ['postprompt'],'lib','python')
sys.path.insert(0,pplib)
from pplib import util

from control.network import Listener
from model.main_model import Model

if __name__ == '__main__':
	print "Host: Starting!"
	util.make_logger('host')
	model = Model(100)
	listener = Listener()
	listener.listen_for_requests(model)
	print "Host: Exiting!"

File: src/ppbackend/Host.java
import pplib.exceptions.PPLoadException;

import ppbackend.control.Listener;
import ppbackend.model.Model;

public class Host{
	public static void main(String[] args){
		try{
			System.out.println("Host: Starting!");
			Model model = new Model(10);
			Listener listener = new Listener(model);
			listener.listenForever();
			System.out.println("Host: Exiting!");
		}
		catch(PPLoadException e){
			System.out.println("Host: Could not load games from database!");
			System.exit(1);
		}
	}
}
